test/extern/spartee/sys-stat-extern.good

```diff
@@ -0,0 +1 @@
+1531167971
```

Comment:
Git doesn't preserve file modification times, so the last modified time of this file will keep changing.  (For example if you checkout master and then check out this branch again, I bet the test will fail.)

Even if it did, the .good file would need updating every time someone modified the .chpl file.  For example, to add a newline at the end.


You could try using `touch -d` to create a file with a known modification time.  Though I'm not sure that's portable to everywhere Chapel runs.  Or what lengths you'd have to go to to be sure it'll result in the same output no matter what timezone the test is run in.

Reply:
I agree with @cassella - there'll have to be some other strategy for this to work out. The most obvious one is to just not print the time result. Another direction would be to create a file and then check its last modified time (with some tolerance).